hngi/MyCustomer
# Included a web notification service feature

---
title: PR - Feature
---

#### What does this PR do?
Includes the web notification service to MyCustomer.

#### Description/outline of task to be completed?
I've included an INTEGRATING.md file with all the descriptions.

#### Specify your testing environment (OS, Hardware etc)
Windows 10 pro 64 bit, Intel Core i3-4005 CPU @ 1.70GHz (4 CPUs) - 4GB RAM  

#### Specify any additional dependencies that this PR introduces
All dependencies can be found in the package.json file

#### How should this be manually tested?
To test, manually, setup a local database and replace the database credentials in the `database/connect.js` file and run `npm start`. The debt table should have at least a cust_name(int), cust_id(int) and a due_date(timestamp) column

#### Any background context you want to provide?
None

#### What issues are fixed by or related to this pull request (mandatory)?
No listed issue. Includes a web notification service feature

#### Screenshots (if appropriate)

#### Prerequisites:
- [ ] I have ran all tests and new and existing unit tests pass locally with my changes
- [ ] I have linted the code using provided linting tools to ensure it meets coding standards
- [ ] I have formatted the code to meet the specified style guide (may require using provided tools)
